src/DataProtection/Abstractions/test/DataProtectionCommonExtensionsTests.cs

```diff
@@ -304,7 +308,8 @@ public void Unprotect_Success()
         mockProtector.Setup(p => p.Unprotect(new byte[] { 0x01, 0x02, 0x03, 0x04, 0x05 })).Returns(new byte[] { 0x48, 0x65, 0x6c, 0x6c, 0x6f });
 
         // Act
-        string retVal = DataProtectionCommonExtensions.Unprotect(mockProtector.Object, "AQIDBAU");
+        string data = "AQIDBAU";
+        string retVal = mockProtector.Object.Unprotect(data);
```

Comment:
What was wrong w/ using `DataProtectionCommonExtensions` explicitly❔ Seems you're making the opposite change in src/DataProtection/Extensions/test/DataProtectionAdvancedExtensionsTests.cs

Reply:
The `DataProtectionAdvancedExtensionsTests` one was my first attempt at fixing the issues before I realized a variable would resolve it, and then I forgot to update that file.

> What was wrong w/ using DataProtectionCommonExtensions explicitly❔

Oh, didn't realize that was already there, I just converted all of them during an intermediate attempt.